starters/apps/qwikcity-test/src/routes/(common)/actions/issue5463/index.tsx

```diff
@@ -19,6 +19,7 @@ export const useDotNotationAction = globalAction$(
           firstName: z.string(),
         }),
       }),
+      someAny: z.any(), // test for making sure it does not add it to the fieldErrors see #6719
```

Comment:
I think we need an explicit test for it in the spec file

Reply:
@wmertens added a test in `server-functions.unit.ts`.

But `expectTypeOf` is pretty weird. I found some issues in vitest and expect-type implementation:
See https://github.com/mmkal/expect-type/issues/31

The any implementation with not seems to work. But I don't know if this will fail at least on build.full.

Let me know what you think.